v-next/hardhat/src/internal/cli/main.ts

```diff
@@ -56,98 +56,80 @@ export async function main(cliArguments: string[], print = console.log) {
     hardhatSpecialArgs.configPath = await resolveHardhatConfigPath();
   }
 
-  try {
```

Comment:
I actually didn't change too much here, just removed the `try/catch` as it's handled in `v-next/hardhat/src/cli.ts`